src/components/Сhoice-of-bikes/ChoiceBikes.jsx

```diff
@@ -0,0 +1,146 @@
+import { Link } from "react-router-dom";
+import "./choiceBikes.scss";
+
+import bike from "../img/Icons/bike.png";
+import material from "../img/Icons/material.png";
+import speed from "../img/Icons/speed.png";
+import tracker from "../img/Icons/tracker.png";
+
+import bicicleta1 from "../../pages/Bicicleta-interna/imgBicicletas/bicicleta1.png";
+import bicicleta2 from "../../pages/Bicicleta-interna/imgBicicletas/bicicleta2.png";
+import bicicleta3 from "../../pages/Bicicleta-interna/imgBicicletas/bicicleta3.png";
+
+import delivery from "./img/delivery.png";
+import quantity from "./img/quantity.png";
+
+import dec from "../img/dec.svg";
+
+const bike1 = "Nimbus Stack";
+const bike2 = "Magic Migth";
+const bike3 = "Mebula Cosmic";
+
+export const BicicletaInterna = ({ img, title}) => {
+  let isMain = img ? "main" : "small";
```

Comment:
Why u use let here?

change variable name
is____ that mean boolean type

Reply:
done